tools/python-deps/setup.py

```diff
@@ -29,7 +29,6 @@
     'pycparser',
     'pyfdt',
     'concurrencytest',
-    'simpleeval',
```

Comment:
You will probably need to increase `version='0.7.1', ` lower in the file so that anyone updating camkes-deps won't keep using the old copy of this file.